miniupnpd/miniupnpd.conf

```diff
@@ -176,7 +176,7 @@ uuid=00000000-0000-0000-0000-000000000000
 # also consider implementing network-specific restrictions
 # CAUTION: failure to enforce any rules may permit insecure requests to be made!
```

Comment:
I would suggest to add here some documentation. Including what special string `NAT-PMP` means.

Reply:
I'd like to fulfill the documentation after this pr get merged.